gui/wms_settings.html

```diff
@@ -1,15 +1,14 @@
 <div class="row">
-    <div class="col-5">
+    <div class="col-7 offset-5">
         <label class="checklabel">
-            <input type="checkbox" id="set_enable_interpolation" checked />
+            <input type="checkbox" data-callback="simpleWmsBodies.setEnableInterpolation" checked />
             <i class="material-icons"></i>
             <span>Interpolate</span>
         </label>
     </div>
-
-    <div class="col-5">
+    <div class="col-7 offset-5">
         <label class="checklabel">
-            <input type="checkbox" id="set_enable_timespan" />
+            <input type="checkbox" data-callback="simpleWmsBodies.setEnableTimeSpan" />
```

Comment:
Is this the time span used for pre-fetching?

Reply:
No, pre-fetching is defined only by the config file. This time-span is used to download a texture over the current time + the time duration defined in config. Practically it request 2 days/months/etc. instead of 1 (for e.g. 2020-04-20/2020-04-21 instead of 2020-04-20 and 2020-04/2020-05 instead of 2020-04) from the WMS server. However it only worked for the Fires dataset for me.